admin_guide/install/deploy_router.adoc

```diff
@@ -241,6 +242,58 @@ To remove a passphrase from a keyfile, you can run:
 # openssl rsa -in <passwordProtectedKey.key> -out <new.key>
 ----
 
+[[using-the-container-network-stack]]
+
+=== Using the Container Network Stack
+
+The OpenShift router runs inside a Docker container and the default behavior is
+to use the network stack of the host (i.e., the node where the router container
+runs). This default behavior has performance benefits as the network traffic
+from remote clients no longer needs to go through a double hop through user
+space to reach the target service and container.
```

Comment:
s/as/because and the phrase "no longer needs to" means we might have to mention that there were previous version that needed that behavior. Also, is "double hop" a standard term? I've not heard it before. Also, hyphenated?

Maybe:

"This default behavior benefits performance, because the network traffic from the remote clients previously needed to go through a double-hop through user space to reach the target service and container."